app/javascript/components/cloud-volume-form/cloud-volume-form.schema.js

```diff
@@ -59,7 +59,7 @@ const createSchema = (fields, edit, ems, loadSchema, emptySchema) => {
         isRequired: true,
         validate: [
           { type: validatorTypes.REQUIRED },
-          async (value) => validateName("cloud_volumes", value)
+          async (value) => validateName("cloud_volumes", value, edit)
```

Comment:
`async(value) => validateName('cloud_volumes', value),`

Reply:
hey @Oferlis , just one more lint error to fix. changing the double quotes for `cloud_volumes` here and they we can merge.

Reply:
hey @Oferlis , just one more lint error to fix. changing the double quotes for `cloud_volumes` here and then we can merge.